DEBUG 2015-04-27 17:15:27,929: 	 Executing command: 'status'
DEBUG 2015-04-27 17:15:27,929: 	 Found /uscms/home/bparida/.crab3 file
DEBUG 2015-04-28 13:44:34,907: 	 Executing command: 'submit'
DEBUG 2015-04-28 13:44:34,907: 	 Found /uscms/home/bparida/.crab3 file
DEBUG 2015-04-28 13:44:34,917: 	 Checking credentials
DEBUG 2015-04-28 13:44:34,918: 	 Found proxy file /tmp/x509up_u47333
DEBUG 2015-04-28 13:44:34,918: 	 Getting proxy life time left
DEBUG 2015-04-28 13:44:35,020: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u47333 -timeleft
 output : 41009

 error: 
 retcode : 0
DEBUG 2015-04-28 13:44:35,123: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u47333 -actimeleft
 output : 41008

 error: 
 retcode : 0
DEBUG 2015-04-28 13:44:35,124: 	 Proxy valid for 11:23:29 hours
INFO 2015-04-28 13:44:35,124: 	 Will use configuration file ZhToBB_crab.py.
DEBUG 2015-04-28 13:44:35,124: 	 Loading configuration
ERROR 2015-04-28 13:44:35,140: 	 Configuration syntax error:
name 'General' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Configuration_syntax_error.
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile.
ERROR 2015-04-28 13:44:35,140: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/cms/crabclient/3.3.15.patch1/bin/crab", line 181, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/cms/crabclient/3.3.15.patch1/bin/crab", line 166, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/cms/crabclient/3.3.15.patch1/lib/python2.6/site-packages/CRABClient/Commands/submit.py", line 39, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args = True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/cms/crabclient/3.3.15.patch1/lib/python2.6/site-packages/CRABClient/Commands/SubCommand.py", line 264, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/cms/crabclient/3.3.15.patch1/lib/python2.6/site-packages/CRABClient/Commands/SubCommand.py", line 82, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Configuration syntax error:
name 'General' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Configuration_syntax_error.
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile.
